src/samplers/support/distributions.jl

```diff
@@ -169,3 +169,38 @@ Distributions.logccdf{T<:Real}(d::Distributions.Truncated, x::T) =
     x <= d.lower ? zero(T) :
     x >= d.upper ? -T(Inf) :
     log(d.ucdf - cdf(d.untruncated, x)) - d.logtp
+
+
+
+
+
+# No info
+immutable Flat <: ContinuousUnivariateDistribution
```

Comment:
It's a flat distribution - so that I can use it as a prior to do maximum likelihood; some Stan models don't specify prior distributions